src/test_fixture/world.rs

```diff
@@ -230,7 +230,8 @@ where
 
         // Perform validation and convert the results we just got: P to O
         let output = join_all(
-            zip(v, m_results).map(|(v, m_result)| async { v.validate(m_result).await.unwrap() }),
+            zip(v, m_results)
+                .map(|(v, m_result)| async move { v.validate(m_result).await.unwrap() }),
```

Comment:
Why did you need to modify this line? Why add this "move"?

Reply:
If got left from changes. Apologies